src/database/seeders/20190704205541-demo-articles.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert(
```

Comment:
Please check my comment on the ratings seeder, you should not be seeding data into your test database this way

Reply:
could you give a better reason for this @tolumide-ng and also suggest a way to go about it?